docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,99 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="linux_command_line"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
```

Comment:
Please mention the exact debian version that was used for the tutorial. We should also mention that is should work for most Linux distros that don't otherwise have a network manager program (or something like that).